src/plugins/open/open.c

```diff
@@ -221,113 +218,112 @@ ipmi_openipmi_send_cmd(struct ipmi_intf * intf, struct ipmi_rq * req)
 			intf->target_addr,intf->target_channel, intf->my_addr);
 
 		if(intf->transit_addr != 0 && intf->transit_addr != intf->my_addr) { 
-		   uint8_t index = 0;
+			uint8_t index = 0;
       
-		   lprintf(LOG_DEBUG, "Encapsulating data sent to "
-			   "end target [0x%02x,0x%02x] using transit [0x%02x,0x%02x] from 0x%x ",
-			   (0x40 | intf->target_channel),
-			   intf->target_addr,
-			   intf->transit_channel,
-			   intf->transit_addr,
-			   intf->my_addr
-			   );      
-
-		   /* Convert Message to 'Send Message' */
-		   /* Supplied req : req , internal req : _req  */
-
-		   if (verbose > 4) {
-		      fprintf(stderr, "Converting message:\n");
-		      fprintf(stderr, "  netfn     = 0x%x\n",  req->msg.netfn );
-		      fprintf(stderr, "  cmd       = 0x%x\n", req->msg.cmd);
-		      if (req->msg.data && req->msg.data_len) {
-			 fprintf(stderr, "  data_len  = %d\n", req->msg.data_len);
-			 fprintf(stderr, "  data      = %s\n",
-				 buf2str(req->msg.data,req->msg.data_len));
-		      }
-		   }
-
-		   /* Modify target address to use 'transit' instead */
-		   ipmb_addr.slave_addr = intf->transit_addr;
-		   ipmb_addr.channel    = intf->transit_channel;
-
-		   /* FIXME backup "My address" */
-		   data_len = req->msg.data_len + 8;
-		   data = malloc(data_len);
-		   if (!data) {
-		      lprintf(LOG_ERR, "ipmitool: malloc failure");
-		      return NULL;
-		   }
-
-		   memset(data, 0, data_len);
-
-		   data[index++] = (0x40|intf->target_channel);
-		   data[index++] = intf->target_addr;
-		   data[index++] = (  req->msg.netfn << 2 ) |  req->msg.lun ;
-		   data[index++] = ipmi_csum(data+1, 2);
-		   data[index++] = 0xFF;    /* normally 0x20 , overwritten by IPMC  */
-		   data[index++] = ( (0)  << 2) | 0 ;           /* FIXME */
-		   data[index++] = req->msg.cmd;
-		   memcpy( (data+index) , req->msg.data, req->msg.data_len);
-		   index += req->msg.data_len;
-		   data[index++] = ipmi_csum( (data+4),(req->msg.data_len + 3)  );
+			lprintf(LOG_DEBUG, "Encapsulating data sent to "
+				"end target [0x%02x,0x%02x] using transit [0x%02x,0x%02x] from 0x%x ",
```

Comment:
Please do alignment with spaces (see [here](https://github.com/ipmitool/ipmitool/wiki/Coding-Standards#indentation-and-alignment)).
Also, this seems to be too deeply indented and crosses 80 characters boundary even with tabsize=4.
If you have time it would be nice to refactor it further and extract some code to helper functions to reduce indentation and improve readability. Or at least break the text line one more time please.

Reply:
I have a patch that fixes a bug in this function, and before I submitted that patch I wanted to attempt at making the function internally consistent and it was using tabs more than spaces.

I'm going to abandon this patchset, and format the entire repo real quick using spaces, etc and submit that for review, then submit my bugfix.